style/grid/main.scss

```diff
@@ -0,0 +1,7 @@
+@import "variables/main.scss";
+@import "mixins/main.scss";
+@import "grid";
+@import "visibility"; // Visibility last for precedence
+
+
+//as for https://github.com/jvdl/sass-flexbox-grid/
```

Comment:
I imported directly this library to have more control on its code, as it is not maintained